config/kubernetes/sig-docs/teams.yaml

```
@@ -83,18 +83,18 @@ teams:
     - yastij
     privacy: closed
   sig-docs-hi-owners:
-    description: Owners for Hindi content
+    description: Approvers for Hindi content
     members:
-    - daminisatya
+    - anubha-v-ardhan
+    - divya-mohan0209
```

```suggestion
    - divya-mohan0209
```

Is she an active contributor? I didn't saw any PR in kubernetes/website repo.

---

Yes, Divya is a very active contributor doing a lot of good work, also recently [nominated for SIG Docs Co-Chair](https://groups.google.com/g/kubernetes-sig-docs/c/AFWmzt43G3Y?pli=1).
We also had an [onofficial poll on #kubernetes-docs-hi Slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1631109069025400?thread_ts=1629833966.493300&cid=CJ14B9BDJ) to which almost almost all of our active contributors including Tim gave +1.

---

Also, Kunal will join as reviewer later once he becomes org member officially.

---

Yes, Divya is a very active contributor doing a lot of good work, she has been involved with SIG-Docs for few years  and was also a Docs shadow for 1.19. She recently got [nominated for SIG Docs Co-Chair](https://groups.google.com/g/kubernetes-sig-docs/c/AFWmzt43G3Y?pli=1) too.

We also had an [unofficial poll on #kubernetes-docs-hi Slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1631109069025400?thread_ts=1629833966.493300&cid=CJ14B9BDJ) to which almost all of our active contributors including Tim gave +1.